src/layouts/AboutLayout.js

```diff
@@ -0,0 +1,338 @@
+  },
+  {
+    icon: "fa-lightbulb",
+    title: "Get inspired",
+    text:
+      "On Climate Connect, you will find blue prints of high-impact projects, that you will be able to replicate at your location. This way you can very easily find the most effective thing you can work on with your specific skillset."
+  },
+  {
+    icon: "fa-eye",
+    title: "Maximum transparency",
+    text:
+      "Our code is available open source and can be found in our Github repository. The impact algorithm will be open source and the community will make the decisions on how to develop it further."
+  }
+];
+
+const members = [
+  {
+    name: "Christoph Stoll",
+    image: "christophstoll.png",
+    location: "Erlangen, Germany"
+  },
+  {
+    name: "Thomas Bove",
+    image: "thomasbove.png",
+    location: "Paris, France"
+  },
+  {
+    name: "Tobias Rehm",
+    image: "tobiasrehm.png",
+    location: "Erlangen, Germany"
+  },
+  {
+    name: "Reece Langerock",
+    image: "reecelangerock.png",
+    location: "Chicago, USA"
+  },
+  {
+    name: "Michael Fischer",
+    image: "michaelfischer.png",
+    location: "Erlangen, Germany"
+  }
+];
+
+function renderBubbles(info) {
+  return info.map(i => {
+    return <InfoBubble data={i} key={i.title}></InfoBubble>;
+  });
+}
+
+function renderMemberInfo(members) {
+  return members.map(m => {
+    return <Member data={m} key={m.name}></Member>;
+  });
+}
+
+export default function about() {
+  return (
+    <Container>
+      <AboutImageContainer>
+        <HeadlineContainer>
+          <Headline>
+            <HeadlineText>CLIMATE CONNECT</HeadlineText>
+          </Headline>
+        </HeadlineContainer>
+      </AboutImageContainer>
+      <h2>
+        We are an international team of volunteers building a non-profit climate action webplatform
+      </h2>
+      <h1>Our goal is to help you fight climate change most effectively</h1>
+      <BubbleGrid>{renderBubbles(info)}</BubbleGrid>
+      <h1>Find out more</h1>
+      <div>
+        <LinkContainer>
+          <h2>
+            <a href="onepager.pdf" target="_blank" rel="noopener noreferrer">
+              <i className="fas fa-file-alt"></i>Read our onepager
+            </a>
+          </h2>
+        </LinkContainer>
+      </div>
+      <div>
+        <LinkContainer>
+          <h2>
+            <a
+              href="xd.adobe.com/view/395518bf-7e51-4eb2-6ff2-3f1b57fdaa17-4f64/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <i className="fas fa-pencil-ruler"></i>View our interactive prototype
+            </a>
+          </h2>
+        </LinkContainer>
+      </div>
+      <div>
+        <LinkContainer>
+          <h2>
+            <Link href="/faq">
+              <a>
+                <i className="fas fa-question"></i>View our FAQ section for in depth explanations
+              </a>
+            </Link>
+          </h2>
+        </LinkContainer>
```

Comment:
These can be combined into a single render function, just with the data in a json or js object